scipy/sparse/linalg/_expm_multiply.py

```diff
@@ -74,12 +127,23 @@ def expm_multiply(A, B, start=None, stop=None, num=None, endpoint=None):
         Number of time points to use.
     endpoint : bool, optional
         If True, `stop` is the last time point.  Otherwise, it is not included.
+    traceA : scalar, optional
+        Trace of `A`. If not given the trace is estimated for linear operators,
+        or calculated exactly for sparse matrices. It is used to precondition
+        `A`, thus an approximate trace is acceptable.
+        For linear operators, `traceA` should be provided to ensure performance
+        as the estimation is not reliable.
```

Comment:
We need a version identifier for the new features.

```suggestion
        as the estimation is not reliable.
        
        .. versionadded:: 1.19.0
```